source/usr/lib/mcvirt/virtual_machine/factory.py

```diff
@@ -118,6 +130,11 @@ def checkName(self, name, ignore_exists=False):
 
         return True
 
+    def checkGraphicsDriver(self, driver):
```

Comment:
Sorry to be a pain, the coding standard specifies that we should be using snake_case for methods (although most of the code in this class pre-exists this and hasn't been converted, so not your fault at all. Just be good to use this for all new code